### DISCUSSION TITLE
One backend to rule them all (multiple fronts)

### DISCUSSION THREAD

**Discussion by timoanttila** [directus/directus#6978] [Feature Requests]
Your software feels great, almost perfect for my special needs, but I would like to use one backend for multiple frontends which means it should be possible to restrict read and write for certain keys / values in some tables. Like teamid = 123456.

Is this possible? I would not like to create a whole frontend for websites' admins as well.

**Comment by timoanttila** [comment]
Well, looks like there is already something like it.

```
{
    "id": {
        "_eq": "$CURRENT_USER"
    }
}
```

Own role for each site and static id for them should do it?

**Reply by rijkvanzanten** [reply]
Yeah, that'll do the job 🙂

**Reply by timoanttila** [reply]
Is it possible to fill a certain field, let's say team, when the current user creates a new page? Because editors don't know the team id which should be a required field. Should I use the role's name or can I use the role's special field, maybe the same name? I could use a relative dropdown if wrong answer can be filtered.

**Reply by rijkvanzanten** [reply]
You can auto-save the current user's ID or Role by using a UUID field type and configuring the "On Create" option:

<img width="622" alt="CleanShot 2021-07-26 at 15 19 12@2x" src="https://user-images.githubusercontent.com/9141017/127046182-cbbab007-4d0d-4227-b127-c8277a4f5582.png">

**Reply by timoanttila** [reply]
Thank you very much. I have finally found the perfect backend of what I have been looking for for years. :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
